build-image.sh

```diff
@@ -74,3 +77,6 @@ kpartx -d $IMAGE
 sha1sum $IMAGE
 
 cp $IMAGE pi-image.img
+
+echo "Creating update package"
+tar --create --xz --file update.tar.xz sb-debs/*.deb
```

Comment:
I'd much rather we use https://github.com/sourcebots/runusb/blob/master/create-update here. It will require a bump for `runusb` to bring the script in though

Reply:
Indeed, I thought about this and decided not to bump `runusb` for it -- I've added a TODO (c522518) and #14 instead.